src/pages/resultsView/enrichments/ExpressionEnrichmentsBoxPlot.tsx

```diff
@@ -0,0 +1,234 @@
+import * as React from 'react';
+import { observer } from "mobx-react";
+import { observable, computed } from 'mobx';
+import styles from "./styles.module.scss";
+import { MolecularProfile, Sample } from 'shared/api/generated/CBioPortalAPI';
+import {
+    ExpressionEnrichmentWithQ,
+    getAlterationsTooltipContent
+} from 'pages/resultsView/enrichments/EnrichmentsUtil';
+import * as _ from "lodash";
+import autobind from 'autobind-decorator';
+import { IBoxScatterPlotPoint, IStringAxisData, INumberAxisData, makeBoxScatterPlotData, getBoxPlotDownloadData } from '../plots/PlotsTabUtils';
+import BoxScatterPlot, { IBoxScatterPlotData } from 'shared/components/plots/BoxScatterPlot';
+import { remoteData, DownloadControls } from 'public-lib';
+import client from 'shared/api/cbioportalClientInstance';
+import LoadingIndicator from 'shared/components/loadingIndicator/LoadingIndicator';
+import { toConditionalPrecision } from 'shared/lib/NumberUtils';
+import { ExtendedAlteration } from '../ResultsViewPageStore';
+import { getSampleViewUrl } from 'shared/api/urls';
+import classNames from "classnames";
+import { getGeneSummary } from '../querySummary/QuerySummaryUtils';
+
+class MRNAEnrichmentsTabBoxPlot extends BoxScatterPlot<IBoxScatterPlotPoint> { }
```

Comment:
lets name this more generically.

since the wrapping component is called BoxPlot, maybe it can be called `EnrichmentsBoxPlotComponent` or something?